### PR TITLE
Organization removal confirmation using name not password

### PR DESCRIPTION
**Pull request by pboguslawski**
Gitea is asking for user password to confirm organization
removal so this operation cannot be done in systems with
SSO authentication (where no user passwords are used).

This mod changes the way gitea confirms organization
removal - user must enter organization name (not user
password) to confirm operation (similar to repository
removal confirmation).

Author-Change-Id: IB#1107219

### CODE REVIEW

**Comment by Kreyren** [comment]
FWIW i prefer the password way in scenarios where I might leave my system unattended (which rarely happens but i like safe-failure)

.. and i prefer the password over name as on GitHub it wastes time assuming password manager filling the password for me.

Proposing to implement this as a configuration per repository instead.

**Comment by pboguslawski** [comment]
> FWIW i prefer the password way in scenarios where I might leave my system unattended (which rarely happens but i like safe-failure)

This PR is not a  nice to have feature but bugfix - without it gitea does not allow to remove organization when SSO is used without password (i.e. with reverse proxy auth only scenario).

If you feel afraid of org removal accidently then use accout without access to org settings - messing with passwords in SSO scenarios seems not practical.

**Comment by pboguslawski** [comment]
> Proposing to implement this as a configuration per repository instead

This PR does not apply to repo removal but org removal. Gitea confirms repo removal using its name so its compatible with both password and SSO auth.

**Comment by Kreyren** [comment]
> This PR is not a nice to have feature but bugfix - without it gitea does not allow to remove organization when SSO is used without password (i.e. with reverse proxy auth only scenario).

Noted

---

> This PR does not apply to repo removal but org removal. Gitea confirms repo removal using its name so its compatible with both password and SSO auth.

My fault then! I guess i confused it with the org removal, would still argue for this to be a configurable option for org and repo.

**Review by noerw — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

File: options/locale/locale_pl-PL.ini
Comment (6543) [inline options/locale/locale_pl-PL.ini]:
```suggestion
```
will be overwritten by CI, pleace localize via Crowdin after pull got merged

**Comment by 6543** [comment]
@pbarsalou pleace change as suggested & update branch

**Review by techknowlogick — APPROVED** [review]

File: options/locale/locale_pl-PL.ini
Comment (pboguslawski) [reply options/locale/locale_pl-PL.ini]: PL translation removed in 36a8157f76e5dc3a4f129a5d1af199e4007b33a4. Let me know if you need `enterred_invalid_org_name` removed from `locale_en-US.ini` also.

Comment (6543) [reply options/locale/locale_pl-PL.ini]: No, eng is source of translation & is good as it is :)

**Comment by 6543** [comment]
@pboguslawski pleace merge master into?

Since your fork is within a org github do not allowvus maintainer to keep your feature branch up to date to merge ...

**Comment by pboguslawski** [comment]
> pleace merge master into?

Just pressed "Update branch" on this PR page.

File: options/locale/locale_pl-PL.ini
Comment (6543) [reply options/locale/locale_pl-PL.ini]: added translation for you here: https://crowdin.com/translate/gitea/68/enus-pl?filter=basic&value=2#q=enterred_invalid_org_name

Comment (pboguslawski) [reply options/locale/locale_pl-PL.ini]: Translation is ok - thank you.

**Pull request merged**
